File: Day 2/arithmetic.rb
_original_price = gets().to_f
_tip = gets().to_i
_tax = gets().to_i
_res = _original_price * (1 + 1.0 * _tax / 100 + (1.0 * _tip / 100))
puts ("The final price of the meal is $#{ _res.round }.")

File: Day 2/arithmetic.java
import java.util.*; //Scanner

public class Arithmetic {

    public static void main(String[] args) {
    
        Scanner sc = new Scanner(System.in);
        double originalPrice = sc.nextDouble();
        int tip = sc.nextInt();
        int tax = sc.nextInt();
        double res = originalPrice + (double)originalPrice * tip / 100 + (double)originalPrice * tax / 100;
        int finalPrice = (int)Math.round(res);
        System.out.println("The final price of the meal is $" + finalPrice + ".");
    }
}